composer.json

```
@@ -33,11 +23,17 @@
     },
     "autoload": {
         "psr-4": {
-            "SilverStripe\\SpamProtection\\": "code/",
+            "SilverStripe\\SpamProtection\\": "src/",
             "SilverStripe\\SpamProtection\\Tests\\": "tests/"
         }
     },
     "license": "BSD-3-Clause",
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+    "config": {
+        "allow-plugins": {
+            "composer/installers": true,
+            "silverstripe/vendor-plugin": true
+        }
+    }
```

```suggestion
    "prefer-stable": true
```
This doesn't belong on modules - only needed on recipes.

---

@GuySartorelli it's added when doing composer install (during testing)

---

Yes, if you install from the repo directly rather than adding it to an existing project it will do that. But it shouldn't be committed to the repository (matching the other supported modules).
We could change that but we should discuss it in the core committers channel and change it across the board if we do change it.